gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -519,10 +594,13 @@ gum_module_enumerate_ranges (const gchar * module_name,
   gum_mach_header_t * header;
   guint8 * p;
   guint cmd_index;
+  gint pid;
```

Comment:
Variable declarations should flow chronologically (except if it represents the return value, in which case it always goes first), so this one should be moved right before `header`'s declaration.